Tools/pyco/pyco.py

```diff
@@ -274,13 +274,8 @@ def rewritten_bytecode(code: types.CodeType, builder: Builder) -> bytes:
         oparg: int
         opcode, oparg = instrs[i : i + 2]
         if opcode == LOAD_CONST:
-            # TODO: Handle EXTENDED_ARG
             if i >= 2 and instrs[i - 2] == EXTENDED_ARG:
```

Comment:
Clearly you can drop the condition, assuming this works.

Reply:
I wasn't sure this is our final state. Are we going to put back immediates, and if so how does that change anything in the 1x1 mapping we currently have for opcodes?

Reply:
We will worry about that in a separate PR, but the idea is to keep dummies in co_consts for indexes where we use immediates (at least until the compiler generates those, which is a separate project). If there's an EXTENDED_ARG + LOAD_CONST combo, we replace the E_A with a NOP.